torch/quantization/fx/graph_module.py

```diff
@@ -47,8 +44,34 @@ def __deepcopy__(self, memo):
         fake_mod.__dict__ = copy.deepcopy(self.__dict__)
         return ObservedStandaloneGraphModule(fake_mod, self.graph)
 
-def mark_observed_standalone_module(module: GraphModule) -> GraphModule:
-    return ObservedStandaloneGraphModule(module, module.graph)
-
 def is_observed_standalone_module(module: Any) -> bool:
     return isinstance(module, ObservedStandaloneGraphModule)
+
+
+class QuantizedGraphModule(GraphModule):
+    def __init__(self, root: Union[torch.nn.Module, Dict[str, Any]], graph: Graph):
+        super().__init__(root, graph)
+
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super()._save_to_state_dict(destination, prefix, keep_vars)
+        for attr_name in dir(self):
+            if "_packed_weight" in attr_name and \
+               isinstance(getattr(self, attr_name), torch._C.ScriptObject):  # type: ignore
+                packed_weight = getattr(self, attr_name)
+                destination[prefix + attr_name] = packed_weight
```

Comment:
You are missing the handling of the `keep_vars` param here I think. If this is a Tensor, it should be `packed_weight if keep_vars else packed_weight.detach()`.

Reply:
`packed_weight` is not a Tensor, it's a TorchBind object, do we still need to call detach() on it?

Reply:
Do they contain Tensors that can require gradients? If so, you should detach all of them yes.

Reply:
`packed_weight` has type ConvPackedParams/LinearPackedParams, and they do not contain any tensors, but they are generated from Tensors (by packing them).